pkg/rke2/psp.go

```diff
@@ -231,6 +230,17 @@ func setPSPs(clx *cli.Context) func(context.Context, daemonsConfig.Control) erro
 					}
 					ns.Annotations[namespaceAnnotationGlobalRestricted] = cisAnnotationValue
 				}
+
+				if _, err := cs.RbacV1().ClusterRoleBindings().Get(ctx, nodeClusterRoleBindingName, metav1.GetOptions{}); err != nil {
+					if !apierrors.IsAlreadyExists(err) {
```

Comment:
For a GET I do not think that this conditional check ever evaluates to anything other than true, therefore you can remove it.